examples/guestbook/README.md

```diff
@@ -199,81 +276,51 @@ If you want to see the logs for a given container, you can run:
 $ docker logs <container_id>
 ```
 
-### Step Two: Fire up the redis master service
+### Step Three: Create the redis slave service
```

Comment:
@bgrant0607 You mention this link. So I think we put the service and RC into the same file and separated by `---`. But example_test.go can't test file which use `---` to separate multi resources. It tests  files with runtime.Object. Should we use list kind instead of separated by `---`?